source/includes/_tipser-react-elements.md

```diff
@@ -197,58 +199,82 @@ import { Store } from '@tipser/tipser-elements';
-for products on sale.
-
-
-### Cart ###
-
-The Cart component with the cart icon can be placed anywhere on your website. (It is highly advisable to place it in your navigation element among other icons such as search, home etc.) However, if you want to keep it visible at all times, attached to the right side of the viewport, you can use these styles:
-
-```css
-.cart-icon {
-    position: fixed;
-    right: 0;
-    top: 121px;
-    background: #fff;
-    padding: 10px;
-    box-shadow: -2px 2px 7px rgba(0,0,0,0.3);
-    z-index: 10;
-}
+The component will be rendered on Your site as shown below:
+[![](checkout_component.png)](/images/checkout_component.png)
+ 
+ 
+## Modular Checkout
+Use this more advanced way of displaying Checkout, if you want to have more control over its parts.
+
+The `Checkout` consists of several react components:
+
+```html
+<Checkout>
+  <CartProducts />
+  <CustomerAddressDelivery />
+  <CartPromoCode />
+  <CartSummary />
+  <CheckoutPayment />
+  <CustomerAddressBilling />
+</Checkout>
 ```
 
-### Adding Primary color ###
-
-If you'd like to unify our design with your own color-theme, you can use our primary-color [configuration option](#primary-color). If your primary color is a bright one, you might want to change also the text color of the elements with primary color background. For example in buttons by default the color we use is white.
-
-```css
+To make them all connected, you need to place them in a wrapper component `<Checkout/>` and pass all the data via props `{...checkout}`:
 
-/* if your primary color is bright,
-    you may consider changing also the text color for elements like buttons: */
-.te-button-text {
-    color: #fff;
-}
+```jsx
+import React from 'react';
+import { Checkout, CheckoutData } from '@tipser/tipser-elements/dist/all';
+import { Route, Switch, withRouter } from 'react-router';
+
+export const CheckoutMultipage = withRouter(({ match }) => (
+  <div>
+    <div className="te-multipage-label">Checkout multipage</div>
+    <Checkout>
+      {(checkout: CheckoutData) => (
+        <Switch>
+          <Route path={`${match.url}/step-1`}>
+            <CheckoutPage1 checkout={checkout} />
+          </Route>
+          <Route path={`${match.url}/step-2`}>
+            <CheckoutPage2 checkout={checkout} />
+          </Route>
+        </Switch>
+      )}
+    </Checkout>
+  </div>
+));
+
+const CheckoutPage1 = ({ checkout }) => (
+  <>
+    <h2>Step 1</h2>
+    <CartProducts {...checkout} />
+    <CustomerAddressDelivery {...checkout} />
+    <CartSummary {...checkout} />
+  </>
+);
+
+const CheckoutPage2 = ({ checkout }) => (
+  <>
+    <h2>Step 2</h2>
+    <CartPromoCode {...checkout} />
+    <CheckoutPayment {...checkout} />
+  </>
+);
 ```
-If you'd like to change other elements' color as well, please use specific classes to override the styles.
+
+The order in which the components appear is arbitrary and depends entirely on your needs and UX design.
```

Comment:
Not sure if I understand what we're trying to say here.

Maybe:

Feel free to position the checkout components in any order that suits your UX needs?